gptel-context.el

```diff
@@ -91,8 +91,10 @@ context chunk.  This is accessible as, for example:
   context.  If there is already a gptel context at point, remove it
   instead.
 
-- If in Dired, add marked files or file at point to the context.
-  With negative prefix ARG, remove them from the context instead.
+- If in Dired, add marked files or file at point to the context. If
```

Comment:
Do you think we should ask for confirmation (`y-or-n-p`) when adding a directory with `gptel-add`?

Reply:
I weakly lean towards asking for confirmation. The prompt could include the number of files to be added.

Reply:
Here’s an attempt: https://github.com/karthink/gptel/pull/438/commits/8e5e3c867131f0c795c7eb62dd8ba28b25a9cf27

Note that the user will be asked once for each directory. This seems like the correct behavior, though it can become tedious if the user is adding several dirs.